core/src/main/java/com/linecorp/armeria/internal/common/AbstractHttp2ConnectionHandler.java

```diff
@@ -111,13 +112,20 @@ protected final void onConnectionError(ChannelHandlerContext ctx, boolean outbou
 
         handlingConnectionError = true;
         if (Exceptions.isExpected(cause) || isGoAwaySentException(cause, connection())) {
-            // Ignore silently.
+            logger.debug("{} HTTP/2 connection error:", ctx.channel(), cause);
```

Comment:
As you know the error is safely ignored. How about setting the level to `trace` to log the log only when users really want to delve into something?

Reply:
Good idea 👍